package.json

```diff
@@ -268,6 +268,7 @@
     "@metamask/chain-agnostic-permission": "^0.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.7.0",
+    "@metamask/design-system-tailwind-preset": "^0.3.0",
```

Comment:
The `@metamask/design-system-tailwind-preset` generates all of the design token based tailwind classnames and helps standardize our tailwind configuration across the organization